### ISSUE TITLE
Cooldown, permission and prefix with space

### ISSUE THREAD

**Issue by johan-perso** [utarwyn/discord-tictactoe#25]
This can be very nice to add in the tictactoe a functionality to set a cooldown for person who make the command.
I also want a fonctionality to make only people with the grade "Premium" can execute the command (but he can dual a person without the grade.
And I have a problem when I made a command with space (like "ac tictactoe", my bot have a space in the prefix so I cannot use the command with the good prefix).

**Comment by utarwyn** [comment]
Hello, thank you for this idea! A very interesting addition :+1:

**Comment by utarwyn** [comment]
Hello @johan-perso! For now I have added a way of providing role identifiers with the configuration object/file. If member which is typing the command does not have admin permission or one of configured allowed roles, it will not be able to start a game. Is it ok for you? 

*Example of configuration object:*
```json
{
  "token": "MY_TOKEN",
  "command": "-tic tac toe",
  "allowedRoleIds": [
    "626454442014801920",
    "374724997035589634"
  ]
}
```

Check new configuration example file here: https://github.com/utarwyn/discord-tictactoe/blob/next/config/config.example.json

I have also added a configuration to define a cooldown between two command executions and space is now supported.
**All of these features will be available in next minor release (v2.1.0).**
